Translations/translation_ZH_TW.json

```diff
@@ -172,35 +172,35 @@
 		},
 		"PowerPulsePower": {
 			"text2": "電源脈衝",
-			"desc": "為保持電源喚醒而通電所用的功率 <watt（瓦特）>"
+			"desc": "為保持電源喚醒而通電所用的功率 (watt[瓦特])"
 		},
 		"HallEffSensitivity": {
 			"text2": "磁場敏感度",
-			"desc": "磁場感應器用作啟動待機模式的敏感度 <關=停用 | 低=最低敏感度 | 中=中等敏感度 | 高=最高敏感度>"
+			"desc": "磁場感應器用作啟動待機模式的敏感度 (關=停用 | 低=最低敏感度 | 中=中等敏感度 | 高=最高敏感度)"
 		},
 		"LockingMode": {
 			"text2": "按鍵鎖定",
-			"desc": "於焊接模式時，同時長按兩個按鍵啟用按鍵鎖定 <無=停用 | 增=只容許增熱模式 | 全=鎖定全部>"
+			"desc": "於焊接模式時，同時長按兩個按鍵啟用按鍵鎖定 (無=停用 | 增=只容許增熱模式 | 全=鎖定全部)"
 		},
 		"MinVolCell": {
 			"text2": "最低電壓",
-			"desc": "每顆電池的最低可用電壓 <伏特> <3S: 3.0V - 3.7V, 4/5/6S: 2.4V - 3.7V>"
+			"desc": "每顆電池的最低可用電壓 (3S: 3 - 3.7V | 4-6S: 2.4 - 3.7V)"

```

Comment:
@alvinhochun
... this one?
If so, what exactly has to be reverted?

thanks in advance
kind regards